certificateshop-logic/src/main/java/com/epam/esm/dao/impl/jdbc/ColumnNames.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.esm.dao.impl.jdbc;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Stores names of the database tables columns, which are used by {@link GiftCertificateExtractor}
+ * {@link CertificateTagMapper} and {@link GiftCertificateMapper} to fetch data from a ResultSet.
+ */
+public class ColumnNames {
+    private ColumnNames() {
+    }
```

Comment:
Move constructor after the below class variables.

Reply:
DEFAULT_PARAMS it not a method. It is the constant.

Question: is it reasonable to move this class to an own new package?

Reply:
You should move the method below the constants.

Reply:
You can keep it here